packages/ui/components/pending-email-tx-row/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+
+import { Text, TouchableHighlight, View, Image } from 'react-native'
+import { connect } from 'react-redux'
+
+import { currencyFormats } from 'lndr/format'
+import UserData from 'lndr/user'
+import InviteTransaction from 'lndr/invite-transaction'
+import { currencySymbols } from 'lndr/currencies'
+import { getUcacCurrency } from 'reducers/app'
+
+import { white, darkAqua } from 'theme/include/colors'
+import style from 'theme/account'
+import general from 'theme/general'
+
+import language from 'language'
+const { inviteLink } = language
+
+interface Props {
+  onPress?: () => void
+  inviteTx: InviteTransaction
+  user: UserData
+  getUcacCurrency: (ucac: string) => string
+  
+}
+
+interface PassedProps extends React.Props<any> {
+  user: UserData
+  inviteTx: InviteTransaction
+  onPress?: () => void
+}
+
+class InviteTransactionRow extends Component<Props> {
+  constructor(props) {
+    super(props)
+  }
+
+  getTitle() {
+    return inviteLink
+  }
+
+  getAmount() {
+    const { inviteTx, getUcacCurrency } = this.props
+    let sign = ''
+
+    if (inviteTx.direction === 'lend') {
+      sign = '+'
+    } else {
+      sign = '-'
+    }
```

Comment:
Are the `+` and `-` symbols universal, or do these need localization?

Reply:
universal